soh/src/overlays/actors/ovl_En_Ossan/z_en_ossan.c

```diff
@@ -601,8 +601,9 @@ void EnOssan_Init(Actor* thisx, GlobalContext* globalCtx) {
         return;
     }
 
-    // Completed Dodongo's Cavern
-    if (this->actor.params == OSSAN_TYPE_BOMBCHUS && !(gSaveContext.eventChkInf[2] & 0x20)) {
+    // gSaveContext.eventChkInf[2] & 0x20 - Completed Dodongo's Cavern
```

Comment:
nitpick, but i think putting this under the "don't kill bombchu shop" line might read a little more clearly

the description of the flag feels like additional context where as the "don't kill bombchu" line is describing the behavior